src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -226,7 +227,7 @@ function ComposerWithSuggestions({
             }
             emojisPresentBefore.current = emojis;
             setIsCommentEmpty(!!newComment.match(/^(\s)*$/));
-            setValue(newComment);
+            setValue(Platform.OS === 'android' ? newComment : convertToLTR(newComment));
```

Comment:
Previously we used this only for android to get the desired result for components that are using convertToLTR function but previously we were not using the function for input box. Now we are doing it for those components as well as for the input box also. But in case of android we don't want the text inside input box to get converted by the converToLTR function because it breaks the input box functionality. This is because Android does not properly support bidirectional text for mixed content due to missing support for writingDirection. So now we want to use convertToLTR function for all platforms including android but we don't want it in case of input box of android. So just for the input box of android we need a check to see whether the platform is android or not.

Reply:
If we make specific file for android in which we return plane text in that case it will apply to all components of android as well for the input box of android also. But we require the function to return plane text just in case of input box of android, we want it to work normally when its used inside other components of android. So to do this I think we can just check the platform when using it inside the input box.

Reply:
Great thank you for the explanation. I think I understand now. However, we are really strict about not using Platform.OS. You said "now we want to use convertToLTR function for all platforms including android but we don't want it in case of input box of android". So let's just use a custom convertToLTR function in an android specific file for ComposerWithSuggestions. Or we could create a separate module convertToLTRForComposer which has an android specific implementation. That's probably best. Also please include a comment about why we need to do this.